verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -107,104 +192,107 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
         const ageRange = values.age
             ? { start: values.age, end: values.age }
             : { start: values.minAge, end: values.maxAge };
-        try {
-            await axios.post('/api/cases', {
-                demographics: {
-                    sex: values.sex,
-                    ageRange: ageRange,
-                    ethnicity: values.ethnicity,
-                    nationalities: values.nationalities,
-                    profession: values.profession,
+        const newCase = {
+            demographics: {
+                sex: values.sex || '',
```

Comment:
why was it needed to add `|| ''` ? I would think it's better to leave it as undefined than to set a value of the empty string. Otherwise doesn't the DB value become empty string rather than not set?

Reply:
that was a mistake and I removed it already actually